runtime/lua/vscode-neovim/defaults.lua

```diff
@@ -2,6 +2,7 @@
 local M = {}
 
 function M.setup()
+  vim.opt.syntax = "on"
```

Comment:
I think this still has to be syntax("on")

Reply:
Yes, I just realized something is wrong:)